src/pkg/selectors/testdata/groups.go

```diff
@@ -4,7 +4,10 @@ import (
 	"github.com/alcionai/corso/src/pkg/selectors"
 )
 
-const TestChannelName = "Test"
+const (
+	TestChannelName = "Test"
+	TestChatTopic   = "Test"
```

Comment:
Curious, why is chat stuff in groups.go?

Reply:
because I was too lazy to make a new file 😆